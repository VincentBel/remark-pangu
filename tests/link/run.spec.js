runSpec(__dirname)
runSpec(__dirname, {
    link: false
})