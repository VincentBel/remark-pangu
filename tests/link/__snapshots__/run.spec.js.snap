// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`link.md with options: {"link":false}: link.md 1`] = `
[说明a说明1](http://example.com/链接a链接1 "标题a标题1")

[说明b说明2](http://example.com/链接b链接2)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[说明 a 说明 1](http://example.com/链接a链接1 "标题a标题1")

[说明 b 说明 2](http://example.com/链接b链接2)

`;

exports[`link.md with options: undefined: link.md 1`] = `
[说明a说明1](http://example.com/链接a链接1 "标题a标题1")

[说明b说明2](http://example.com/链接b链接2)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[说明 a 说明 1](http://example.com/链接a链接1 "标题 a 标题 1")

[说明 b 说明 2](http://example.com/链接b链接2)

`;
